app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -348,6 +362,33 @@ export default class SampleViewV2 extends React.Component {
     return parsedValue;
   };
 
+  adjustMetricPrecision = species => {
+    Object.entries(species).forEach(([countType, metricValue]) => {
+      if (countType in METRIC_DECIMAL_PLACES) {
+        species[countType] = parseFloat(metricValue).toFixed(
+          METRIC_DECIMAL_PLACES[countType]
+        );
+      } else if (countType === "nt") {
+        Object.entries(species.nt).forEach(([countType, metricValue]) => {
```

Comment:
Here we know it is `metricKey`. We should also not reuse the variable we defined on the outer loop.

Reply:
Gotcha! So you mean replacing `countType` with `metricKey` right?